test/testoauth.cpp

```diff
@@ -52,6 +54,24 @@ class FakePostReply : public QNetworkReply
             emit metaDataChanged();
             emit finished();
             return;
+        } else if (redirectToPolicy) {
+            setHeader(QNetworkRequest::LocationHeader, "/my.policy");
+            setAttribute(QNetworkRequest::RedirectionTargetAttribute, "/my.policy");
+            setAttribute(QNetworkRequest::HttpStatusCodeAttribute, 307);
```

Comment:
would a `302` make a difference here? As far as I understood per default bigip will use 302

Reply:
it doesn't make a difference but i updated the PR anyway:)